### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][ColourContrast][MAS1.4.3][IntelliJ][Wipro]: Colour contrast ratio is below 4.5:1 for selected created web app in Run webapp  window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1383]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Create a new project.
3. Navigate to and select created Project ->Azure->Run on web app.
4. "Run on web app" dialog will open.
5. Tab to and select "Use Existing web app\ Create Web app" radio button.
6. Tab to choose a web app and select the created web app and observe the colour contrast ratio for web apps and artifacts under "Before launch" expand collapse button .

**Expected Result:**
Colour contrast ratio should be greater than or equal to 4.5:1  for selected created web app.

**Actual Result:**
 Colour contrast ratio is  4.4:1 for selected created web app in Run web app  window.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)
**CCA version**: 2.5.0.0

![cca for created web app](https://user-images.githubusercontent.com/22956130/35681679-0486ccb8-0713-11e8-9158-0dea36cbe936.png)

**Comment by Eskibear** [comment]
It has external dependency, IntelliJ IDEA handles its own theme, refer to:
https://youtrack.jetbrains.com/issue/IDEA-132299

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749854
